lib/thrift/parser/nimble.ex

```diff
@@ -0,0 +1,189 @@
+defmodule Thrift.Parser.Nimble do
```

Comment:
Do you think we'll have multiple parsecs in here? Otherwise, just `Thrift.Parser.Lexer` (`lib/thrift/parser/lexer.ex`) seems better.